docs/coordinates/satellites.rst

```diff
@@ -96,14 +101,33 @@ For example, to find the overhead latitude, longitude, and height of the satelli
 
 Or, if you want to find the altitude and azimuth of the satellite from a particular location:
 
+.. note ::
+    In this example, the intermediate step of manually setting up a topocentric `~astropy.coordinates.ITRS`
+    frame is necessary in order to avoid the change in stellar aberration that would occur if a direct
+    transform from geocentric to topocentric coordinates using ``transform_to`` was used. Please see
+    the documentation of the `~astropy.coordinates.ITRS` frame for more details.
+
 .. doctest-requires:: sgp4
 
     >>> from astropy.coordinates import EarthLocation, AltAz
     >>> siding_spring = EarthLocation.of_site('aao')  # doctest: +SKIP
-    >>> aa = teme.transform_to(AltAz(obstime=t, location=siding_spring))  # doctest: +IGNORE_WARNINGS
+    >>> topo_itrs_repr = itrs_geo.cartesian.without_differentials() - siding_spring.get_itrs(t).cartesian  # doctest: +IGNORE_WARNINGS
+    >>> itrs_topo = ITRS(topo_itrs_repr, obstime = t, location=siding_spring)  # doctest: +IGNORE_WARNINGS
+    >>> aa = itrs_topo.transform_to(AltAz(obstime=t, location=siding_spring))  # doctest: +IGNORE_WARNINGS
     >>> aa.alt  # doctest: +FLOAT_CMP
-    <Latitude 10.95229446 deg>
+    <Latitude 10.94799670 deg>
     >>> aa.az  # doctest: +FLOAT_CMP
-    <Longitude 59.30081255 deg>
+    <Longitude 59.28803392 deg>
+
+For a stationary observer, velocity in the `~astropy.coordinates.ITRS` is independent of location,
+so if you want to carry the velocity to the topocentric frame, you can do so as follows:
+
+.. doctest-requires:: sgp4
+
+    >>> itrs_geo_p = itrs_geo.cartesian.without_differentials()  # doctest: +IGNORE_WARNINGS
```

Comment:
Are there actually warnings emitted for any of these lines?

Reply:
Don't know. This was just copy/paste from the original, but I can try it.

Reply:
This worked.